src/mixins/filing-template-mixin.ts

```diff
@@ -10,6 +10,7 @@ import { ActionTypes, AssociationTypes, CorrectionErrorTypes, EffectOfOrders, Fi
   RoleTypes } from '@/enums/'
 import { CorpTypeCd } from '@bcrs-shared-components/corp-type-module/'
 import { StaffPaymentOptions } from '@bcrs-shared-components/enums/'
+import { StartDate } from '@/components/common'
```

Comment:
I believe this mixin doesn't need this visual component.

Reply:
Yes it has been removed already